test/test_transforms.py

```diff
@@ -15,6 +15,11 @@
 if IMPORT_SCIPY:
     import scipy
 
+import pytest
+import numpy as np
```

Comment:
``import numpy as np`` should be under the ``if IMPORT_LIBROSA`` right?

Reply:
Yes, I'll fix that and add a skipif for the tests